app/services/album.js

```diff
@@ -0,0 +1,42 @@
+const https = require('https');
+const url = 'https://jsonplaceholder.typicode.com/albums';
+
+const getAll = () =>
+  new Promise((resolve, reject) => {
+    https.get(url, (res, err) => {
```

Comment:
Use a package like [this](https://github.com/request/request-promise) for this, also catch errors and map to an internal error